SADeprecationWarning on test

```
$ pip install -U SQLAlchemy
$ pytest tests/test_databese.py
```

```
_________________________________________________ TestDataBase.test_setup_default_db __________________________________________________

self = <tests.test_database.TestDataBase object at 0x7f8c948656a0>

    def test_setup_default_db(self):
        from chainerui.database import db
        db.init_db()
>       assert db.setup(url=None)

tests/test_database.py:36:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
chainerui/database.py:61: in setup
    echo=echo
venv/fixdep/lib/python3.6/site-packages/sqlalchemy/engine/__init__.py:455: in create_engine
    return strategy.create(*args, **kwargs)
venv/fixdep/lib/python3.6/site-packages/sqlalchemy/engine/strategies.py:95: in create
    dialect = dialect_cls(**dialect_args)
venv/fixdep/lib/python3.6/site-packages/sqlalchemy/dialects/sqlite/pysqlite.py:321: in __init__
    SQLiteDialect.__init__(self, **kwargs)
venv/fixdep/lib/python3.6/site-packages/sqlalchemy/dialects/sqlite/base.py:1433: in __init__
    default.DefaultDialect.__init__(self, **kwargs)
<string>:2: in __init__
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

fn = <function DefaultDialect.__init__ at 0x7f8c9675f730>
args = (<sqlalchemy.dialects.sqlite.pysqlite.SQLiteDialect_pysqlite object at 0x7f8c96dce7b8>,)
kwargs = {'case_sensitive': True, 'convert_unicode': True, 'dbapi': <module 'sqlite3.dbapi2' from '/home/tanakad/.pyenv/versions/3.6.7/lib/python3.6/sqlite3/dbapi2.py'>, 'empty_in_strategy': 'static', ...}
m = 'convert_unicode'

    @decorator
    def warned(fn, *args, **kwargs):
        for m in check_defaults:
            if kwargs[m] != defaults[m]:
                warnings.warn(
>                   messages[m], exc.SADeprecationWarning, stacklevel=3
                )
E               sqlalchemy.exc.SADeprecationWarning: The create_engine.convert_unicode parameter and corresponding dialect-level parameters are deprecated, and will be removed in a future release.  Modern DBAPIs support Python Unicode natively and this parameter is unnecessary.

venv/fixdep/lib/python3.6/site-packages/sqlalchemy/util/deprecations.py:122: SADeprecationWarning
```

--- a/chainerui/database.py
+++ b/chainerui/database.py
@@ -56,7 +56,6 @@ class Database(object):
         try:
             engine = create_engine(
                 url,
-                convert_unicode=True,
                 connect_args=connect_args,
                 echo=echo
             )

